src/decisionengine_modules/graphite_client.py

```diff
@@ -21,7 +22,8 @@ class Graphite:
     def __init__(self, host="fifemondata.fnal.gov", pickle_port=2004):
         self.graphite_host = host
         self.graphite_pickle_port = pickle_port
-        self.logger = structlog.getLogger()
+        self.logger = structlog.getLogger(CHANNELLOGGERNAME)
+        self.logger = self.logger.bind(module=__name__.split(".")[-1], channel="")
```

Comment:
Is this CHANNELLOGGERNAME or DELOGGERNAME or something else?
This is not inheriting from Module, I'm not sure it is running in a channel process.
If you think it is then it is OK, it will inherit the channel name from the logger definition using
```python
self.logger = self.logger.bind(module=__name__.split(".")[-1])
```
Otherwise, we should check w/ Shreyas.